CHANGELOG.md

```diff
@@ -10,9 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add compression support when sending and receiving sdp messages.
+- Add `rawMetricReport` property to `DefaultClientMetricReport` to store raw [`RTCStatsReport`](https://developer.mozilla.org/en-US/docs/Web/API/RTCStatsReport) and expose it via `metricsDidReceive` event.
```

Comment:
Feel like this function can just be named `rtcStatsReport()`? `raw/underlying` prefix could work but i don't think its necessary. @ltrung what do you think.

Reply:
+1

Reply:
Renamed `rawMetricReport` to `rtcStatsReport`.